full-service/Cargo.toml

```diff
@@ -67,7 +67,7 @@ serde = { version = "1.0", default-features = false, features = ["alloc", "deriv
 serde-big-array = "0.4.1"
 serde_derive = "1.0"
 serde_json = { version = "1.0", features = ["preserve_order"] }
-structopt = "0.3"
+clap = { version = "4.0", features = ["derive", "env"] }
```

Comment:
Nit: we sort dependencies in an alphabetical order

Reply:
fixed